users/ninjonas/README.md

```diff
@@ -17,7 +17,7 @@ See: https://docs.qmk.fm/#/feature_userspace
 - [Kyria](../../keyboards/kyria/keymaps/ninjonas)
 
 ## Features
-### [Keys](ninjonas.h#L37)
+### [Keys](ninjonas.h)
 |Code | Description |
 |---|---|
 |K_LOCK | MacOS shortcut to execute lock command  + CTRL + Q |
```

Comment:
The  character doesn't render on all systems (mine among them!); if you replace all instances of it in this file with `&#8984;`, it should render clearly in more environments.

![image](https://user-images.githubusercontent.com/18669334/74596943-08292d00-500b-11ea-82eb-3638f9ef8979.png)

Reply:
thank you!

Reply:
This should be reverted IMO. `&#8984;` (which is the Apple logo, not eth) will only render on macOS, and changing it to an HTML entity won't do anything. In fact, it will now fail to render inside code tags.

Reply:
Yes, it was only `users/ninjonas/README.md` that needed this change. The keymap readme files have this inside code blocks, so it just prints `&#8984;` instead of the Place of Interest sign. Perhaps those should just use `Cmd`.

Reply:
Thanks for the review. I'm totally cool with reverting. just let me know what's the recommended action.